module/VuFind/src/VuFind/Controller/SearchController.php

```diff
@@ -88,7 +88,8 @@ public function editmemoryAction()
             }
         } elseif ($removeFacet) {
             $defaults = ['operator' => 'AND', 'field' => '', 'value' => ''];
-            extract($removeFacet + $defaults);
+            extract($removeFacet);
+            extract($defaults);
```

Comment:
This changes the meaning of the code -- it will cause the defaults to always override the incoming values. I think it probably makes sense to totally rewrite this as:

<pre>
$query = $initialParams->removeFacet(
    $removeFacet['field'] ?? '',
    $removeFacet['value'] ?? '',
    $removeFacet['operator'] ?? 'AND'
);
</pre>

That seems a lot more straightforward.

Reply:
I amended the commit

Reply:
Great, I'll just drop this commit the next time I rebase, since all the remaining parts have already been merged to dev.

Reply:
Thanks!